default/scripting/techs/defense/BarrierShield.focs.txt

```diff
@@ -108,7 +114,8 @@ Tech
                 (LocalCandidate.LastTurnAttackedByShip < CurrentTurn - 1)
             ]
             priority = [[DEFAULT_PRIORITY]]
-            effects = SetMaxShield value = Value + 150 accountinglabel = "DEF_PLAN_BARRIER_SHLD_4"
+            effects = SetMaxShield value = Value + 150
```

Comment:
these should be left on the same line; `accountinglabel` is a paremter to the effect